Docs: Add multi-tenant query curl examples (#6530)
grafana/loki: docs/sources/api/_index.md

  }
}
```

> Set the `x-scope-orgid` header,

Reviewer: ```suggestion
If your Loki cluster has multi-tenancy enabled, set the `x-scope-orgid` header,
```

> Set the `x-scope-orgid` header,

Reviewer 2: ```suggestion
Set the `X-Scope-OrgID` header,
```
Header field names should be case-insensitive but this is how we use it everywhere.

as defined in [Grafana Loki Multi-Tenancy](../operations/multi-tenancy/),

> to query specific tenants.

Reviewer: ```suggestion
to specify to Loki which specific tenant you want to query.
```

> Here is the same example query applied to the single tenant called `Tenant1`:

Reviewer: For some reason I don't like the word applied but I can't come up with a better alternative so I think its fine. I guess I think of the query as being 'directed at' that tenant, rather I think of 'applied' as more referring to something more permanent (e.g., you apply a setting or apply a configuration change).

Reviewer 2: Maybe
```
Here is the same example query for the single tenant called `Tenant1`:
```

```bash

> curl -H 'x-scope-orgid:Tenant1' \

Reviewer 2: Nit picking
```suggestion
curl -H 'X-Scope-OrgID: Tenant1' \
```

  -G -s "http://localhost:3100/loki/api/v1/query" \
  --data-urlencode \
  'query=sum(rate({job="varlogs"}[10m])) by (level)' | jq
```

> To query against the three tenants `Tenant1`, `Tenant2`, and `Tenant3`,

Reviewer: ```suggestion
If you have cross tenant query federation enabled (link to docs), you can query multiple tenants at once. 

To query against the three tenants `Tenant1`, `Tenant2`, and `Tenant3`,
```

Author: The sentence above this one contains the link to the discussion on enabling this, so I don't want to repeat the prose or the link.

specify the tenant names separated by the pipe (`|`) character:

```bash

> curl -H 'x-scope-orgid:Tenant1|Tenant2|Tenant3' \

Reviewer 2: ```suggestion
curl -H 'X-Scope-OrgID: Tenant1|Tenant2|Tenant3' \
```

  -G -s "http://localhost:3100/loki/api/v1/query" \
  --data-urlencode \
  'query=sum(rate({job="varlogs"}[10m])) by (level)' | jq
```

> The same Grafana Enterprise Logs example query against the three tenants

Reviewer: ```suggestion
The same Grafana Enterprise Logs example query against the three tenants requires the use of Basic Authentication.  
```

Reviewer 2: This documentation is for Loki OSS right? Do we reference GEL there? I don't know the process well enough.

Author: Yes, @jeschkies. In the short term, we're including a GEL example in the Loki docs, so folks know that it is different for GEL. Our GEL readers have to find their info on this page.

> includes the tenant names as users, separated by the pipe (`|`) character.

Reviewer: ```suggestion
Set the username to a concatenated list of all the tenant names, with each name separated by the pipe (`|`) character.
```

> The password in this example is access policy token that has been

Reviewer 2: ```suggestion
The password in this example is an access policy token that has been
```

defined in the `API_TOKEN` environment variable:

```bash

> curl -u Tenant1|Tenant2|Tenant3:$API_TOKEN \

Reviewer 2: ```suggestion
curl -u "Tenant1|Tenant2|Tenant3:$API_TOKEN" \
```

}
```

If your cluster has 

> [Grafana Loki Multi-Tenancy](../operations/multi-tenancy/) enabled,

Author: It does feel stilted and weird.  However, a style guideline is to make the link name the same as the web page title or subsection title that we're sending someone to.  It helps readers know that they've landed at the link we intended for them. That web page is titled "Grafana Loki Multi-tenancy," so I used it as the link title.

I'm going to leave this for now, but happy to figure out a future change that rewords and still follows the style guideline.
